app.rb

```diff
@@ -0,0 +1,96 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_people
+    puts 'List of People:'
+    @people.each { |person| puts "ID: #{person.id}, #{person.class.name}: #{person.name}" }
+  end
+
+  def list_books
+    puts 'List of Books:'
+    @books.each_with_index { |book, index| puts "#{index}. #{book.title} by #{book.author}" }
+  end
+
+  def create_person
+    puts "Enter the person's name:"
+    name = gets.chomp
+
+    puts "Enter the person's age:"
+    age = gets.chomp.to_i
+
+    puts 'Is the person a student? (yes/no):'
+    is_student = gets.chomp.downcase == 'yes'
+
+    @people << (is_student ? Student.new(name, age) : Teacher.new(name, age))
```

Comment:
- [ ] Great job working on this milestone. While creating a person, your app give two options (Yes/No) Taking No means the person being created is not a student. When I take No, I have an error and the app exits as you can see on the screenshot below. Kindly review and fix this issue.
![image](https://github.com/berylcl/OOP-school-library/assets/62849872/5104cd41-9167-49c5-ac6b-14281f725e62)